.openhands/microagents/repo_eng_agent.md

```
@@ -0,0 +1,55 @@
+
+## Review Scope (When Triggered)
+The agent will review the following context:
+- Open Pull Requests
+- Open Issues
+- Comments on Issues or PRs
+- Commits and associated code changes
+- All currently labelled Issues/PRs
+
+## Core Capabilities
+
+### Bug Identification and Fixing
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)